src/Styles/StyledComponents.js

```diff
@@ -1,6 +1,9 @@
 import styled from "styled-components";
-//NavBar
 
+// colors
+const darkerGrey = "#34373a";
```

Comment:
![image](https://user-images.githubusercontent.com/1214752/58742288-df132e80-83e0-11e9-9421-67ecf7dd9100.png)
Not sure how much I like the dark grey. I know we haven't chosen a color scheme, but the site needs to be brighter IMO. The text on the image is also hard to read/focus on. Is there another image we could choose or some style we could add to make it pop out? Is that a free-use image as well?

Reply:
I updated this to a lighter color scheme